### PR TITLE
implemet bubble sort algorithm

### PR DESCRIPTION
**Pull request by Jeko0**

### CODE REVIEW

File: week6_bubble_sort/lib/buble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          sorted = true
+        end
+      end
+      break unless sorted
+    end
+    @arr
+  end
+
+  def input_validation
+    raise ArgumentError, "invalid input" unless @arr.is_a?(Array) && @arr.all? { |element| element.is_a?(Numeric) }
```
Comment (alexshvedko) [inline week6_bubble_sort/lib/buble_sort.rb]: long line

Comment (Jeko0) [reply week6_bubble_sort/lib/buble_sort.rb]: fixed long line problem

File: week6_bubble_sort/lib/buble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  def call
+    input_validation
+    implement_buble_sort
+  rescue ArgumentError => e
+    e.message
+  end
+
+  private
+
+  def implement_buble_sort
```
Comment (alexshvedko) [inline week6_bubble_sort/lib/buble_sort.rb:19]: nested logic, bad practise

File: week6_bubble_sort/lib/buble_sort.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    size = @arr.length
+
+    loop do
+      sorted = false
+      (size - 1).times do |i|
+        if @arr[i] > @arr[i + 1]
+          @arr[i], @arr[i + 1] = @arr[i + 1], @arr[i]
```
Comment (alexshvedko) [inline week6_bubble_sort/lib/buble_sort.rb]: nested logic, it's a bad practise

**Pull request merged**